internal/server/server.go

```diff
@@ -38,7 +39,7 @@ func New(logger Logger, client Client, templates map[string]*template.Template,
 	mux.Handle("/javascript/", static)
 	mux.Handle("/stylesheets/", static)
 
-	return http.StripPrefix(prefix, securityheaders.Use(mux))
+	return otelhttp.NewHandler(http.StripPrefix(prefix, securityheaders.Use(mux)), "sirius-workflow")
```

Comment:
Suggestion for clarity: 
```suggestion
	return otelhttp.NewHandler(http.StripPrefix(prefix, securityheaders.Use(mux)), "supervision-workflow")
```